skxray/img_proc/mathops.py

```diff
@@ -0,0 +1,465 @@
+# Module for the BNL image processing project

```

Comment:
I removed everything except for logical_subtract and logical_nand, which are not included in numpy.